content/en/docs/concepts/services-networking/service.md

```diff
@@ -873,6 +873,10 @@ There are other annotations to manage Classic Elastic Load Balancers that are de
 
         service.beta.kubernetes.io/aws-load-balancer-extra-security-groups: "sg-53fae93f,sg-42efd82e"
         # A list of additional security groups to be added to the ELB
+
+        service.beta.kubernetes.io/aws-load-balancer-target-node-labels: "ingress-gw,gw-name=public-api"
+        # A comma-separated list of key-value pairs which will be used
```

Comment:
nit: A comma separated list of key-value pairs which are used

Reply:
nit: Could you add a new line before line 869?

Reply:
Both fixed!